cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Summary.tsx

```diff
@@ -277,27 +279,29 @@ const Summary: React.FC<StepComponentProps> = ({
             verified
           />
         )}
-        {activeButton === 0 && (
-          <LabelText
-            label={`Amount ${coinAbbr.toUpperCase()}`}
-            text={
-              !maxSend
-                ? `${batchRecipientData[0].amount} ~( $${formatDisplayAmount(
-                    parseFloat(batchRecipientData[0].amount || '0') *
-                      parseFloat(coinPrice),
-                    2,
-                    true
-                  )}) `
-                : `${sendTransaction.sendMaxAmount} ~( $${formatDisplayAmount(
-                    parseFloat(sendTransaction.sendMaxAmount) *
-                      parseFloat(coinPrice),
-                    2,
-                    true
-                  )}) `
-            }
-            verified
-          />
-        )}
+        {activeButton === 0 &&
+          (triggeredBy !== TriggeredBy.WalletConnect ||
+            parseInt(batchRecipientData[0].amount, 10) !== 0) && (
```

Comment:
The condition here and in `cysync/src/pages/mainApp/sidebar/wallet/send/formStepComponents/Recipient.tsx` Line `843` does not match.

Reply:
the recipient file can directly use the value provided by wallet connect but after that, we have to rely on on batchRecipientData[0] for value

Reply:
I just added this same condition in all three files